platform/core/src/utils/urlUtil.js

```diff
@@ -55,11 +55,17 @@ const replaceParam = (path = '', paramKey, paramValue) => {
   return path;
 };
 
+const isValidPath = path => {
+  const paramPatternPiece = `/${PARAM_PATTERN_IDENTIFIER}`;
+  return path.indexOf(paramPatternPiece) < 0;
+};
+
 const queryString = {
   getQueryFilters: getFilters,
 };
 
 const paramString = {
+  isValidPath: isValidPath,
```

Comment:
Same with this: https://eslint.org/docs/rules/object-shorthand

Reply:
ok